packages/opentelemetry-instrumentation-langchain/tests/test_langgraph.py

```
@@ -46,13 +46,13 @@ def calculate(state: State):
     assert openai_span.attributes[SpanAttributes.LLM_REQUEST_TYPE] == "chat"
     assert openai_span.attributes[SpanAttributes.LLM_REQUEST_MODEL] == "gpt-4o"
     assert (
-        openai_span.attributes[f"{SpanAttributes.LLM_PROMPTS}.0.content"]
+        openai_span.attributes[f"{GenAIAttributes.GEN_AI_PROMPT}.0.content"]
```

Ruff (0.12.2)

Undefined name `GenAIAttributes`

(F821)